docs/content_credentials.md

```diff
@@ -0,0 +1,96 @@
+# Content Credential Proposal
+
+## Background
+We have long struggled with the need to save and later continue to edit C2PA manifests.
+The spec doesn't call out a specific way to do this. It only allows for appending new signed manifests.
+But there are many scenarios where we need a work in progress.
+- Validating an ingredient once and saving it for later use.
+- Frequent small edits over time without creating a long chain of signed manifests
+- Applications that want to keep track of changes over time and export later.
+
+We created the concept of a Builder archive to help address this. But the archive is a new format
+that needs to be documented and supported over time as content changes.
+
+## Proposal
+A new proposal is that we always store data as standard JUMBF c2pa manifest stores.
+
+The same format can be used for signed manifests, working stores and saved ingredients. 
+
+- in binary JUMBF (application/c2pa) format
+- the JUMBF may be embedded in a file, in the cloud or in a sidecar.c2pa 
+- We MAY want to allow the JUMBF manifest to not be signed or associated with an asset.
+
+
+## Goals
+
+1) Provide a way to save validated ingredients so they can be added to Builders later.
+2) Archive the current state of a Builder and restore it later to continue work.
+
+For Ingredients - you can create a Builder and add ingredients to it. This builder can then be signed and saved either embedded in content or standalone as a .c2pa. Later you can use Reader to read the saved asset. Any ingredient in the Reader can be added to a Builder using builder.add_ingredient_from_reader(reader, ingredient_uri); // note there should be a shorthand for using the parent ingredient.
+
+For Builder archives - Sign and save a manifest, either embedded or sidecar.  If you want to continue editing a saved manifest, then use Reader to read it, and then use reader.to_builder() (or Builder::from_reader?()) to convert it back into an editable manifest.
+
+- Note that this will work when reading any content. You can always extract an ingredient to a Builder.
+
+- You MAY not always be able to convert a reader into a Builder (TBD - do we need limits on this?)
+
+- The signature here can be local and does not need to be on a trust list. The purpose is to be able to detect any tampering of the data. But you could also use a trusted signature.
+
+
+- You can store the c2pa data as a .c2pa sidecar without modifying the source. The dest file can be a data sink.
+
+- signing the bits of the source image and capturing a thumbnail are optional.
+
+### new methods
+
+```
+    let builder = reader.to_builder()?;
+
+    builder.add_ingredient_from_reader(&reader)?;
+```
+
+### Questions
+
+- Should we allow any Reader to be converted to a Builder, or only those with the same claim_generator? Maybe there is some other flag.
```

Comment:
Services may need to gather things from multiple claim generators? If multiple services do something, each with their claim generator, and sends it to another for final signing?